index.js

```diff
@@ -1362,7 +1362,29 @@
             };
           };
         };
-        return S.encase3(f, x, y, z);
+        return encase3(f, x, y, z);
+      });
+
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Takes a value of any type and a Maybe, and returns an Either.
+  //. If the second argument is a Nothing, a Left containing the first
```

Comment:
Might not be worth it for the sake of consistency throughout the docs, but when the types of arguments are different, it is easier for me to follow if the type is referenced rather than the argument position, ie

`If the Maybe is a Nothing, a Left containing the first argument is returned.`

Obviously since `is a Nothing` immediately follows `second argument`, I should know it is a Maybe I'm dealing with, but when I hit "second argument" I always pause.. think "second argument..??" jump back to the signature.. then continue on :)